acp/acp_local.go

```diff
@@ -152,26 +239,56 @@ func (l *ACPLocal) VerifyAccessRequest(
 	resourceName string,
 	docID string,
 ) (bool, error) {
-	checkDocResponse, err := l.localACP.GetQueryService().VerifyAccessRequest(
-		l.localACP.GetCtx(),
-		&types.QueryVerifyAccessRequestRequest{
-			PolicyId: policyID,
-			AccessRequest: &types.AccessRequest{
-				Operations: []*types.Operation{
-					{
-						Object:     types.NewObject(resourceName, docID),
-						Permission: permission.String(),
-					},
-				},
-				Actor: &types.Actor{
-					Id: actorID,
+	// FIXME remove once Identity is refactored
```

Comment:
todo: @Lodek any todos merged into the Defra repo must be linked to a github issue.  If/when creating one for this, please make sure it contains more info than this comment currently does :)  Atm it is unclear as to why it is temporary/can be removed after the refactor.

Reply:
:saluting_face: Is there an issue for the work you were doing on refactoring the Identity to accept the DID form the JWS Shahzad?

Reply:
We can sync up on a call. The JWS stuff was done already you would just need to generate dids and plug them in now.